Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -690,18 +692,35 @@ lemma coeff_one_mul (φ ψ : R⟦X⟧) : coeff R 1 (φ * ψ) =
     mul_comm, add_comm]
   norm_num
 
-/-- First coefficient of the `n`-th power of a power series with constant coefficient 1. -/
-lemma coeff_one_pow (n : ℕ) (φ : R⟦X⟧) (hC : constantCoeff R φ = 1) :
-    coeff R 1 (φ ^ n) = n * coeff R 1 φ := by
+/-- First coefficient of the `n`-th power of a power series. -/
+lemma coeff_one_pow (n : ℕ) (hn : 1 ≤ n) (φ : R⟦X⟧) :
```

Comment:
It seems that theorem holds if n = 0 (for ridiculous reasons) : both sides are zero.

Reply:
Even better then!